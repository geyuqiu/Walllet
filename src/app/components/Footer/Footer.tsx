import React from "react";

export const Footer = () => (
	<footer className="bg-gray-light flex items-center justify-center p-4 fixed bottom-0 w-full z-100 text-black-lightest text-sm">
		<span>2020 &copy; ARK.io | All rights reserved</span>
	</footer>
);
