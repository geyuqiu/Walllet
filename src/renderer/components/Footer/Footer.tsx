import React from "react";

export const Footer = () => (
	<footer className="bg-gray-100 p-6 min-h-10vh">
		<div className="flex items-center">
			<span>2020 &copy; ARK.io | All rights reserved</span>
		</div>
	</footer>
);
