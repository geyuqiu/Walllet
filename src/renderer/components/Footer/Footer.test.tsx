import { render } from "@testing-library/react";
import React from "react";
import { MemoryRouter } from "react-router-dom";

import { Footer } from "./Footer";

describe("Footer", () => {
	it("should match snapshot", () => {
		const { container } = render(
			<MemoryRouter>
				<Footer />
			</MemoryRouter>
		);
		expect(container).toMatchSnapshot();
	});
});
