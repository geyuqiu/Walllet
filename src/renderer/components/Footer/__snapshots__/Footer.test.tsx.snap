// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<div>
  <footer
    class="warning min-h-10vh flex items-center justify-center"
  >
    <span>
      2020 Â© ARK.io | All rights reserved
    </span>
  </footer>
</div>
`;
